### DISCUSSION TITLE
Crash in restricted region

### DISCUSSION THREAD

**Discussion by HikaruIchijyo** [z-huang/music#193] [General]
there is a crash simple app close i can try to reproduce & send u a video but cant logs. since i dont have acess to adb.

i am in a country where ytm requires vpn i guess
Steps:
search anything click on 1st name(artist in all tab)
select play radio & boom crashes.

if anyone have any proxy url(not any vpn or client specificaly only for this app) plz share

device: redmi note 10s
android 12

i am not sure if this should be in issues thats y i put it here also i am not sure if this is due to country restriction of feature not implemented.

**Comment by z-huang** [comment]
I don't have this issue.

**Reply by HikaruIchijyo** [reply]
need video or something?

btw do u hsve any proxy url?

**Reply by z-huang** [reply]
I need adb log.

> btw do u hsve any proxy url?

Nope.

**Reply by HikaruIchijyo** [reply]
> I need adb log.
> 
> > btw do u hsve any proxy url?
> 
> Nope.

that i cant give cuz i dont have access to adb stuff for many days

**Reply by z-huang** [reply]
> i dont have access to adb stuff

Why? You just need a computer.

**Reply by HikaruIchijyo** [reply]
> I need adb log.
> 
> > btw do u hsve any proxy url?
> 
> Nope.

can u implement log catcher or something like tachiyomi implementations??

**Reply by z-huang** [reply]
Yes, I will take a look at how NewPipe does this.

> tachiyomi implementations

What's this?

**Reply by owenconnorz** [reply]
Cloudstream has the logcat section definitely handy to have on this app to 😄

**Reply by HikaruIchijyo** [reply]
> > tachiyomi implementations
> 
> What's this?

https://github.com/tachiyomiorg/tachiyomi

**Reply by HikaruIchijyo** [reply]
> > i dont have access to adb stuff
> 
> Why? You just need a computer.

i literally dont have access to pc these days

**Reply by z-huang** [reply]
Oh, that's okay.

**Comment by HikaruIchijyo** [comment]
> I don't have this issue.

ok i can confirm it only crashes for restricted countried(ytm)

got my point?
it doesnt crash if i use vpn so something like fetching from ytm crashes it if u r from country where ytm is not available

**Reply by z-huang** [reply]
I tried a Korean VPN and found that only YouTube Premium users can use YouTube Music. So, this app is not usable in regions that have this restriction. Even if you can search in the app, you can't play songs.

**Reply by HikaruIchijyo** [reply]
> I tried a Korean VPN and found that only YouTube Premium users can use YouTube Music. So, this app is not usable in regions that have this restriction. Even if you can search in the app, you can't play songs.

but the issue is why crashing if it doesnt waana play

anyway to bypass it?

**Reply by z-huang** [reply]
It can be fixed.

**Reply by z-huang** [reply]
Fixed.

**Reply by z-huang** [reply]
Fixed.
